sys/stdio/stdio.c

```diff
@@ -78,3 +78,10 @@ int stdio_available(void)
     return tsrb_avail(&stdin_isrpipe.tsrb);
 }
 #endif
+
+#if IS_USED(MODULE_STDIO_CLEAR_RX)
+void stdio_clear_rx(void)
```

Comment:
Since this is only linked in when it's being used, you can drop the pseudo-module